### PR TITLE
Add February RubyGems Update

### PR DESCRIPTION
**Pull request by lauragift21**
Hi @deivid-rodriguez can you help me review this draft to ensure I captured work done in February?

Thank you!

### CODE REVIEW

File: _posts/2022-03-04-february-rubygems-updates.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+The following is a non-exhaustive list of the improvements included in the above releases (see changelogs for more further information):
+
+- resolved some long-standing issues with our CI workflow and worked on some long-standing configuration issues - [#5324](https://github.com/rubygems/rubygems/pull/5324).
+- fixed an issue with a corrupt lockfile when you run `bundle check` and have to re-resolve locally - [#5344](https://github.com/rubygems/rubygems/pull/5344).
+- fixed typo in multiple gemfiles warning - [#5342](https://github.com/rubygems/rubygems/pull/5342).
+- added clarification for `bundle-config` `"with"` option  - [#5346](https://github.com/rubygems/rubygems/pull/5346).
+- the `BUNDLE_WITH` and `BUNDLE_WITHOUT` environment variables shouldn't be persistent locally - [#5335](https://github.com/rubygems/rubygems/pull/5335).
```
Comment (deivid-rodriguez) [inline _posts/2022-03-04-february-rubygems-updates.md:20]: I would tweak this item to 

```suggestion
- Fix issue with `BUNDLE_WITH` and `BUNDLE_WITHOUT` environment variables being silently persisted locally - [#5335](https://github.com/rubygems/rubygems/pull/5335).
```

And we should also add an important change made this month:

```
- `bundle config` now saves configuration locally by default when run inside an application context - [#4512](https://github.com/rubygems/rubygems/pull/4152).
```

Other than that looks good!

Comment (lauragift21) [reply _posts/2022-03-04-february-rubygems-updates.md]: Got it! Thanks for the suggestion. Will add this change.

File: _posts/2022-03-04-february-rubygems-updates.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+In February, Rubygems gained [45 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2022-02-01%7D...master@%7B2022-02-31%7D), contributed by 9 authors. There were 252 additions and 160 deletions across 58 files.
+
+## RubyGems.org News
+
+This month, RubyGems.org saw several bug fixes and updates, some of which include the following:
+
+- enabled **Rails 6.1** default in `application.rb` - [#2966](https://github.com/rubygems/rubygems.org/pull/2966)
+- disabled `mfa_required_since` usage - [#2965](https://github.com/rubygems/rubygems.org/pull/2965)
+- fixed exploded elasticsearch import - [#2963](https://github.com/rubygems/rubygems.org/pull/2963)
```
Comment (simi) [inline _posts/2022-03-04-february-rubygems-updates.md:30]: this was reverted for now

Comment (lauragift21) [reply _posts/2022-03-04-february-rubygems-updates.md:30]: Hi @simi, all of it or just the #2966?

Comment (simi) [reply _posts/2022-03-04-february-rubygems-updates.md:30]: #2963

Comment (lauragift21) [reply _posts/2022-03-04-february-rubygems-updates.md:30]: got it thanks!

**Pull request merged**